src/codegen/impl_debug.rs

```diff
@@ -184,28 +186,40 @@ impl<'a> ImplDebug<'a> for Item {
                     // The simple case
                     debug_print(name, quote! { #name_ident })
                 } else {
-                    // Let's implement our own print function
+                    if ctx.options().use_core {
+                        Some((
+                            format!("{}: [...]", name), vec![]
+                            ))
```

Comment:
nit: indentation is off.

Reply:
I don't see how, it seems to match the rest of the style to me. The old code gained an indent level of 4 spaces due to the if, and the new code mirrors that.

Reply:
The parenthesis are four spaces to the right:

```
                        Some((
                            format!("{}: [...]", name), vec![]
                            ))
```

Should be:

```
                        Some((
                            format!("{}: [...]", name), vec![]
                        ))
```

Same below for `Vector`. In any case it's not a huge deal.